src/User.php

```diff
@@ -213,6 +228,27 @@ protected function getConsistencyValidationErrors()
             }
         }
 
+        //  Validate new password has not been previously used
+        if ($this->passwordChanged && AuthenticationSettings::singleton()->compareNewUserPasswordWithPreviousEntries) {
+
+            $numberOfPastPasswordsToCompareTo = AuthenticationSettings::singleton()->numberOfPastPasswordsToCompareTo;
```

Comment:
I think we could maybe combine the flag setting and numerical setting into one - with zero being disabled. I think it might confused people who just discover the numerical setting, set it but don't see the feature work.

Reply:
This setting has now been removed and replaced with the property numberOfPastPasswordToCompareTo to control both scenarios